eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -1,168 +1,116 @@
-          parameters:
-            SparseCheckoutPaths:
-              - "sdk/${{ parameters.ServiceDirectory }}/**/*.json"
-              - '!sdk/**/test-recordings/*'
-              - '!sdk/**/session-records/*'
-            JobTemplatePath: /eng/pipelines/templates/jobs/live.tests.yml
-            AdditionalParameters:
-              PreSteps:
-                - ${{ parameters.PlatformPreSteps }}
-                - ${{ parameters.PreSteps }}
-              PostSteps:
-                - ${{ parameters.PlatformPostSteps }}
-                - ${{ parameters.PostSteps }}
-              EnvVars: ${{ parameters.EnvVars }}
-              MaxParallel: ${{ parameters.MaxParallel }}
-              BuildInParallel: ${{ parameters.BuildInParallel }}
-              TimeoutInMinutes: ${{ parameters.TimeoutInMinutes }}
-              Location: ${{ parameters.Location }}
-              ServiceDirectory: ${{ parameters.ServiceDirectory }}
-              TestResourceDirectories: ${{ parameters.TestResourceDirectories }}
-              TestSetupSteps: ${{ parameters.TestSetupSteps }}
-              TestMode: ${{ parameters.TestMode }}
-              TestVersionSupport: ${{ parameters.TestVersionSupport }}
-              Artifacts: ${{ parameters.Artifacts }}
-              AdditionalModules: ${{ parameters.AdditionalModules }}
-              TestResultsFiles: ${{ parameters.TestResultsFiles }}
-              DisableAzureResourceCreation: ${{ parameters.DisableAzureResourceCreation }}
-              TestGoals: ${{ parameters.TestGoals }}
-              TestOptions: ${{ parameters.TestOptions }}
-              UseHttpFaultInjector: ${{ parameters.UseHttpFaultInjector }}
-
-            MatrixConfigs:
-              # Enumerate platforms and additional platforms based on supported clouds (sparse platform<-->cloud matrix).
-              - ${{ each config in parameters.MatrixConfigs }}:
-                -  ${{ config }}
-              - ${{ each config in parameters.AdditionalMatrixConfigs }}:
-                -  ${{ config }}
-            MatrixFilters:
-              - ${{ each cloudFilter in cloud.value.MatrixFilters }}:
-                - ${{ cloudFilter }}
-              - ${{ parameters.MatrixFilters }}
-            MatrixReplace:
-              - ${{ each cloudReplace in cloud.value.MatrixReplace }}:
-                - ${{ cloudReplace }}
-              - ${{ parameters.MatrixReplace }}
-            CloudConfig:
-              SubscriptionConfiguration: ${{ cloud.value.SubscriptionConfiguration }}
-              SubscriptionConfigurations: ${{ cloud.value.SubscriptionConfigurations }}
-              Location: ${{ coalesce(parameters.Location, cloud.value.Location) }}
-              Cloud: ${{ cloud.key }}
+extends:
+  template: /eng/pipelines/templates/stages/1es-redirect.yml
+  parameters:
+    stages:
+      - template: /eng/pipelines/templates/stages/archetype-sdk-tests-isolated.yml
+        parameters: ${{ parameters }}
```

Comment:
TIL. This is massive. I laboriously wrote this crap out.

Reply:
Yeah I wish I had figured it out before you finished python

Reply:
Otherwise I would have had to duplicate the sdl config like 20 times in Java too